pkg/controllers/vdb/podfacts.go

```diff
@@ -426,6 +431,8 @@ func (p *PodFacts) genGatherScript(vdb *vapi.VerticaDB, pf *PodFact) string {
 		df --block-size=1 --output=size %s | tail -1
 		echo -n 'localDataAvail: '
 		df --block-size=1 --output=avail %s | tail -1
+		echo -n 'admintoolsExists: '
+        which admintools &> /dev/null && echo true || echo false
```

Comment:
Can you fix the alignment with this line?

Reply:
It's weird, I didn't have this one on my local machine

Reply:
ah tabs issues, just fix it!